From ac8b8af3a814453e50474025bb19eeef981e7c12 Mon Sep 17 00:00:00 2001
From: Lennard Hofmann <lennard.hofmann@web.de>
Date: Tue, 15 Jun 2021 14:43:42 +0000
Subject: [PATCH] make: remove misplaced dolar sign

If LDFLAGS is set to "foo", the build will fail because the command "ccfoo" does not exist (notice there is no space between "cc" and "foo")
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index b0abc4b..846a265 100644
--- a/Makefile
+++ b/Makefile
@@ -11,7 +11,7 @@ OBJ=lswt.o wlr-foreign-toplevel-management-unstable-v1.o xdg-output-unstable-v1.
 GEN=wlr-foreign-toplevel-management-unstable-v1.c wlr-foreign-toplevel-management-unstable-v1.h xdg-output-unstable-v1.c xdg-output-unstable-v1.h
 
 lswt: $(OBJ)
-	$(CC)$ $(LDFLAGS) -o $@ $(OBJ) $(LIBS)
+	$(CC) $(LDFLAGS) -o $@ $(OBJ) $(LIBS)
 
 $(OBJ): $(GEN)
 
-- 
2.32.0

